letsencrypt/client/apache/obj.py

```diff
@@ -58,7 +58,7 @@ class VirtualHost(object):
 
     """
 
-    def __init__(self, filep, path, addrs, ssl, enabled, names=None):
+    def __init__(self, filep, path, addrs, ssl, enabled, names=None):  # pylint: disable=too-many-arguments
```

Comment:
maybe the limit is a bit low if it complains about 5/6 args and the limit should be increased a bit.